### PR TITLE
Display cluster related resources in tabs

### PR DESCRIPTION
**Pull request by maciaszczykm**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*: Fixes https://github.com/kubermatic/dashboard/issues/2780

**Special notes for your reviewer**: 

Design:
98645324-143fe200-2332-11eb-8b3f-665ad8f604fc

Implementation:
Cluster-gracious-almeide-in-Project-default-

**Release note**:

```release-note
Cluster related resources will be now displayed in tabs.
```

### CODE REVIEW

**Comment by maciaszczykm** [comment]
/assign @floreks 

PTAL

File: src/app/shared/components/event-card/event-card.component.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<!--
```
Comment (floreks) [inline src/app/shared/components/event-card/event-card.component.html:1]: Please use the new naming scheme

**Comment by floreks** [comment]
Nice. I like how easily we can reuse it and update other places. :tada:

**Comment by maciaszczykm** [comment]
I've updated file names.

@kgroschoff Can you take a look?

**Comment by kgroschoff** [comment]
I'd say lgtm, but sorry for the merge conflict 😅

**Comment by maciaszczykm** [comment]
@kgroschoff Done :)

**Pull request merged**